src/components/cart.js

```diff
@@ -78,10 +82,30 @@ export default class Cart extends Component {
   get lineItemsHtml() {
     return this.lineItemCache.reduce((acc, lineItem) => {
       const data = Object.assign({}, lineItem, this.options.viewData);
+      const fullPrice = data.variant.priceV2.amount * data.quantity;
+      const formattedPrice = formatMoney(fullPrice, this.moneyFormat);
+      const discountAllocations = data.discountAllocations;
+
+      const {discounts, totalDiscount} = discountAllocations.reduce((discountAcc, discount) => {
+        const targetSelection = discount.discountApplication.targetSelection;
+        if (LINE_ITEM_TARGET_SELECTIONS.indexOf(targetSelection) > -1) {
```

Comment:
Small nitpick - we could use `array.includes` here 😄

Reply:
It's not supported natively in IE11 and Safari 8, and we weren't bringing in the core-js polyfill so I opted to used this instead. I don't really have a preference though

Reply:
Ahh boo - I thought we were bringing in the array polyfills 🤔 
I'm not set on using `includes` (I just find it easier to read) but it might be worthwhile bringing in the array polyfills if we currently not (we can do that separately too I guess)